cmd/check/main.go

```diff
@@ -302,3 +309,14 @@ func retryRoundTripper(logger lager.Logger, rt http.RoundTripper) http.RoundTrip
 		RoundTripper: rt,
 	}
 }
+
+func setClientCert(registry string, list []ClientCertKey) []tls.Certificate {
+	var clientCert []tls.Certificate
+	for _, r := range list {
+		if r.Domain == registry {
+			certKey, _ := tls.X509KeyPair([]byte(r.Cert), []byte(r.Key))
```

Comment:
Probably shouldn't drop this error - I could see someone easily messing up the key or cert.